src/Form/OchaAiChatChatForm.php

```diff
@@ -107,48 +107,6 @@ public function getPageTitle(?bool $popup = NULL): TranslatableMarkup {
   public function buildForm(array $form, FormStateInterface $form_state, ?bool $popup = NULL): array {
     $defaults = $this->ochaAiChat->getSettings();
 
-    // Display the form instructions.
-    if (!empty($defaults['form']['instructions']['value'])) {
-      $hide = $form_state->getValue([
-        'instructions',
-        'content',
-        'hide',
-      ], $this->database
-        ->select('ocha_ai_chat_preferences', 't')
-        ->fields('t', ['hide_instructions'])
-        ->condition('t.uid', $this->currentUser->id())
-        ->execute()
-        ?->fetchField()
-      );
-
-      $form['instructions'] = [
-        '#type' => 'details',
-        '#prefix' => '<div id="ocha-ai-chat-instructions" class="ocha-ai-chat-chat-form__instructions">',
-        '#suffix' => '</div>',
-        '#title' => $this->t('Instructions'),
-        '#open' => !empty($hide) ? FALSE : TRUE,
-        '#tree' => TRUE,
-      ];
-      $form['instructions']['content'] = [
-        '#type' => 'processed_text',
-        '#text' => $defaults['form']['instructions']['value'],
-        '#format' => $defaults['form']['instructions']['format'],
-      ];
-      $form['instructions']['content']['hide'] = [
```

Comment:
We lose the ability to hide the instructions. To revisit maybe if people complain when they start testing.